### PR TITLE
Add PVS_proxy module to Network_interface

### PR DESCRIPTION
**Pull request by robhoes**
This module contains metadata and functions to configure a PVS proxy
daemon running on the host.

Signed-off-by: Rob Hoes rob.hoes@citrix.com

### CODE REVIEW

File: network/network_interface.ml [hunk 1]
```diff
@@ -302,3 +302,35 @@ module Bridge = struct
...
+
+	type t = {
+		farm_uuid: string;
+		farm_name: string;
+		servers: Server.t list;
+		clients: Client.t list;
+		vdi: string;
+	}
+	
+	external configure_farm : debug_info -> PVS_proxy.t -> unit = ""
```
Comment (johnelse) [inline network/network_interface.ml:334]: Shouldn't this be `t` rather than `PVS_proxy.t` ?

Comment (robhoes) [inline network/network_interface.ml:334]: You would be right if this were a normal ml file. However, xcp-idl transforms it in creative ways, such that this appears to be necessary.

Comment (robhoes) [inline network/network_interface.ml:334]: For similar reasons I had to put the exception outside the module...

Comment (johnelse) [inline network/network_interface.ml:334]: Fair enough :)

Comment (jonludlam) [inline network/network_interface.ml:334]: What we should do is move to something less crazy... a ppx-based ocaml-rpc perhaps? :-)

Comment (robhoes) [inline network/network_interface.ml:334]: Too late, it's in now :p

**Pull request merged**